airbyte-integrations/connectors/destination-mssql-v2/src/main/kotlin/io/airbyte/cdk/load/data/sql/AirbyteTypeToSqlType.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data.sql
+
+import io.airbyte.cdk.load.data.AirbyteType
+import io.airbyte.cdk.load.data.ArrayType
+import io.airbyte.cdk.load.data.ArrayTypeWithoutSchema
+import io.airbyte.cdk.load.data.BooleanType
+import io.airbyte.cdk.load.data.DateType
+import io.airbyte.cdk.load.data.IntegerType
+import io.airbyte.cdk.load.data.NumberType
+import io.airbyte.cdk.load.data.ObjectType
+import io.airbyte.cdk.load.data.ObjectTypeWithEmptySchema
+import io.airbyte.cdk.load.data.ObjectTypeWithoutSchema
+import io.airbyte.cdk.load.data.StringType
+import io.airbyte.cdk.load.data.TimeTypeWithTimezone
+import io.airbyte.cdk.load.data.TimeTypeWithoutTimezone
+import io.airbyte.cdk.load.data.TimestampTypeWithTimezone
+import io.airbyte.cdk.load.data.TimestampTypeWithoutTimezone
+import io.airbyte.cdk.load.data.UnionType
+import io.airbyte.cdk.load.data.UnknownType
+import java.sql.Types
+
+/** CDK pipeline [AirbyteType] to SQL [Types] converter. */
+class AirbyteTypeToSqlType {
+
+    /**
+     * Converts an [AirbyteType] to the associated SQL [Types] value.
+     *
+     * @param airbyteSchema The stream's Airbyte schema, represented as an [AirbyteType]
+     * @return The associated SQL [Types] value.
+     * @throws IllegalArgumentException if the [AirbyteType] is not supported.
+     */
+    fun convert(airbyteSchema: AirbyteType): Int {
+        return when (airbyteSchema) {
+            is ObjectType -> Types.BLOB
+            is ArrayType -> {
```

Comment:
arrays (and union / unknown) should also just be blob (or equivalent)

Reply:
@edgao The existing MSSQL destination (and old CDK classes) did not support either.  Happy to make them BLOB if that is what they should be.  Maybe I was looking at the wrong code, but I figured that we would start with existing parity.

Reply:
I have changed the mapping to use blob for these types.

Reply:
Missed union...will add that now.

Reply:
Done.